website/front_matter/prize_structure.qmd

```diff
@@ -0,0 +1,51 @@
+# Prize Structure
+
+**Overview**: _The author explains the size and nature of the Genetic Evolution Tournament's prize pool. Next, the author details the prize structure of the Genetic Evolution Tournament and outlines the reasoning for this._
+
+**Disclaimer**: _Suggestions are very much appreciated on the prize structure._
+
+---
+
+Until further notice, the _Genetic Evolution Tournament_ maintains a prize pool of **15K USD**.
+
+There exists some possibility that more funding might become available in the future, so percentages are given for prize quantities.
+
+This prize pool is allocated over several categories: {short-term, medium-term, long-term} X {commenting-prize, forecasting-accuracy-prize}.
+
+One purpose of the tournament is to generate scenarios and discussion surrounding futures involving human enhancement via DNA and reproductive technologies; as such, comments supporting this end are incentivized via commenting prizes.
+
+Another purpose of the tournament is to quantify uncertainty on and predict the occurrence of certain milestones in human enhancement via the aforementioned technologies; as such, accurate and truthful forecasting is incentivized via forecasting prizes.
+
+* Short-Term Insights (35% = 5.25K USD)
+  * Period: Years 1-5, i.e. 2025-2029
+  * Commenting: 10, 0.2K USD prizes
```

Comment:
How much of the prize pool should be allocated to comments in total and then also how much of this total across each category?

Should commenting prizes be uniform (e.g 10, 200 USD prizes) or tiered (1 500 USD, 2 200 USD, etc...)? 

For context, the OWID tournament has tiered comments prizes wheras the GiveWell series has uniform prize allocations. 

In the author's experience, the distribution of comment quality is not so fat-tailed that the top comment is _that_ much greater than the second or third highest-quality comments; hence, the author leans more towards uniformly sized commenting prizes or prizes that only slightly deviate in size. 

Current thinking: 2000 USD per category (there are 3 categories). Around 5-10 commenting prizes. Possibility: per category, 1st - 500 USD, 2nd - 300 USD, then next 12 receive 100 USD.